atlasdb-jepsen-tests/src/main/java/com/palantir/atlasdb/jepsen/events/InfoEvent.java

```diff
@@ -40,7 +40,7 @@
 
     public abstract long time();
 
-    public abstract Optional<String> value();
+    public abstract Optional<Object> value();
```

Comment:
Definitely agree - getting in to the weeds of how the nemeses work is _not_ worth it. Cool, let's leave it as `Object` then :)